[FIX] awesome_owl: Fix missing css variables

clbr-odoo opened the pull request:
Some imports are missing in the manifest causing some warnings and the css doesn't load properly.
Fix was fixed in the master branch, backporting it in 19.0 for the onboarding classes that always happen in the lastest stable.

[See](https://github.com/odoo/tutorials/blame/2eef68071d786c91520ac9e25b514577730df142/awesome_owl/__manifest__.py#L32)

task-none

clbr-odoo committed:
[FIX] awesome_owl: Fix missing css variables

Some imports are missing in the manifest causing some warnings and
the css doesn't load properly.
Fix was fixed in the master branch, backporting it in 19.0
for the onboarding classes that always happen in the lastest stable.

task-none

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/tutorials/pull/1037.png)](https://mergebot.odoo.com/odoo/tutorials/pull/1037)

clbr-odoo committed:
[FIX] awesome_dashboard: Get rid of the deprecated warning

`json` routes were deprecated to `jsonrpc` in 19.0, let's
get rid of the warning to avoid confusion for the newdoos.

task-none

AntoineVDV reviewed (approved):
@robodoo r+

robodoo commented:
@clbr-odoo @AntoineVDV because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward

AntoineVDV commented:
@robodoo rebase-ff

robodoo commented:
Merge method set to rebase and fast-forward.
